pyaerocom/web/aerocom_evaluation.py

```diff
@@ -522,6 +522,7 @@ def compute_json_files_from_colocateddata(self, coldata, obs_name,
                                               model_name):
         """Creates all json files for one ColocatedData object"""
         vert_code = self.get_vert_code(obs_name, coldata.meta['var_name'][0])
+        web_iface_name = self.obs_config[obs_name]['web_interface_name']
```

Comment:
Consider adding try/except block in case `web_iface_name` is not defined, e.g.:
```python
try:
   web_iface_name = self.obs_config[obs_name]['web_interface_name']
except: 
   web_iface_name = obs_name
```
?

Reply:
Good point. I've added this.